vignettes/gDRtestData.Rmd

```diff
@@ -8,22 +8,123 @@ vignette: >
   %\VignetteEncoding{UTF-8}
 ---
 
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
 ```{r, echo=FALSE}
 library(gDRtestData)
 ```
 
 # Overview
 
-The `gDRtestData` package is intended to store and generate example data that can be used through gDR.
+The `gDRtestData` package is intended to store and generate example data that can be used through the `gDR` suite.
 
 # Use Cases
 
-## Data generation
-All key functions can be found in `gDRcore/R/generate_wrappers.R`. In `inst/scripts` you can find a script `generate_example_data.R` showing how to generate and process testing data.
+## Synthetic data generation
+
+Since `gDR`is a computational suite for drug response data from any experiment, a synthetic dataset is also needed for testing and exploration purposes. 
```

Comment:
```suggestion
Since gDR is a computational suite for drug response data from any experiment, a synthetic dataset is also needed for testing and exploration. 
```

Reply:
Done.